tests/test_gcloud_store.py

```diff
@@ -104,9 +106,9 @@ def test_gcstore_pickling(store):
     assert store.get("key1") == b"value1"
 
 
-def test_gcstore_pickling_attrs():
+def test_gcstore_pickling_attrs(gc_credentials):
     store = GoogleCloudStore(
-        credentials="path_to_json",
+        credentials=gc_credentials,
```

Comment:
Why was this change necessary?

Reply:
The `fsspec` filesystem was created in the initializer, which requires correct credentials. I deferred it to the first access now, which is what the previous implementation did.